src/PolyhedralGeometry/Serialization.jl

```diff
@@ -1,170 +1,211 @@
+using JSON
+
 ##############################################################################
 """
-    save_cone(Cone, String)
+    save(C::Cone, filename::String)
 
 Save a cone to a file in JSON format. The first argument is the cone, the
 second argument is the filename.
 """
-function save_cone(C::Cone, filename::String)
+function save(C::Cone, filename::String)
    bigobject = pm_object(C)
    Polymake.save_bigobject(bigobject, filename)
 end
 
 """
-    load_cone(String)
+    load(as::Type{Cone}, filename::String)
 
 Load a cone stored in JSON format, given the filename as input.
 """
-function load_cone(filename::String)
-   bigobject = Polymake.load_bigobject(filename)
-   typename = Polymake.type_name(bigobject)
-   if typename != "Cone<Rational>"
-      throw(ArgumentError("Loaded object is not of polymake type Cone, it has type " * typename))
-   end
-   return Cone(bigobject)
+function load(as::Type{Cone}, filename::String)
```

Comment:
```suggestion
function load(::Type{Cone}, filename::String)
```

Reply:
Thanks, makes sense, wasn't clear to me.